.travis.yml

```diff
@@ -5,6 +5,15 @@ _prepare_deploy: &_prepare_deploy
     - prepare_everything
     - docker pull $PLANET_TEST
     - docker pull $DOCKER_DB_INIT_TEST
+_prepare_multiarch: &_prepare_multiarch
+  before_script:
+    - sudo snap install yq
```

Comment:
why would we run `snap` instead of `apt`
-> specially since there is a travis directive to use apt fast

Reply:
because the `yq` processor is not available on apt. Its only available on the new Snapcraft.